data/json/monsters/power_leech.json

```diff
@@ -0,0 +1,188 @@
+[
+  {
+    "id": "mon_leech_blossom",
+    "type": "MONSTER",
+    "name": "leech blossom",
+    "description": "A resplendent alien fern, crowned with flowers colored deep indigo.  It appears to be the centerpiece of this otherworldly bloom.",
+    "default_faction": "nether",
+    "species": [ "LEECH_PLANT" ],
+    "volume": "92500 ml",
+    "weight": "40 kg",
+    "hp": 100,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "K",
+    "color": "light_cyan",
+    "aggression": 100,
+    "morale": 100,
+    "armor_bash": 15,
+    "vision_day": 30,
+    "vision_night": 30,
+    "luminance": 200,
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 5,
+        "gun_type": "emp_frond",
+        "fake_skills": [ [ "gun", 3 ], [ "rifle", 3 ] ],
```

Comment:
Rifle skill is useless here, as emp frond uses pistol skill? The same for other users of this gun.